Dockerfile

```diff
@@ -1,4 +1,14 @@
 ARG  SERVERCORE_VERSION
 
+FROM golang:1.16-nanoserver-1809 AS build-env
```

Comment:
could you switch to `rancher/golang:1.16-windowsservercore`, we build our own windows golang containers with a manifest pointing to the 5 different versions so you dont have to specify -1809 and the build servers can negotiate their version.